.travis.yml

```diff
@@ -8,8 +8,10 @@ osx_image: xcode6.4
 
 env:
   matrix:
-    
-    - CONDA_PERL=5.22.2.1
+    - CONFIG=osx_python2.7
+    - CONFIG=osx_python3.5
+    - CONFIG=osx_python3.6
```

Comment:
@dopplershift b/c this is using `python` to activate the VCs we cannot merge this right now or we'll create bogus matrix items like these :-(